fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java

```diff
@@ -155,9 +155,9 @@
 
     private static final String FHIR_SERVER_NAME = "IBM Watson Health Cloud FHIR Server";
     private static final String FHIR_COPY_RIGHT = "(c) Copyright IBM Corporation 2018, 2019";
-    private static final String EXTENSION_URL = "http://ibm.com/watsonhealth/fhir/extension";
-    private static final String BASIC_RESOURCE_TYPE_URL = "http://ibm.com/watsonhealth/fhir/basic-resource-type";
-    private static final String REQUEST_HEADER_URL = "http://www.ibm.com/watsonhealth/fhir/extensions/http-request-header";
+    private static final String EXTENSION_URL = "http://ibm.com/watson/health/fhir/extension";
+    private static final String BASIC_RESOURCE_TYPE_URL = "http://ibm.com/watson/health/fhir/basic-resource-type";
```

Comment:
What is this one used for?

Reply:
EXTENSION_URL is used in the addExtensionElements for the Capability Statement

BASIC_RESOURCE_TYPE_URL
doSearch > createBasicCodeSearchParameter uses it as a valueSystem for the 'code' type.
*** this is for virtual resources 

Not sure about REQUEST_HEADER_URL

```
// If we see an entry for the "http-request-header" URL, then parse the value.
                    if (REQUEST_HEADER_URL.equals(url)) {
```

Reply:
BASIC_RESOURCE_TYPE_URL and REQUEST_HEADER_URL should get deleted.  We can do that now or in a separate issue / PR.

Reply:
I suggest a separate Issue.  do you want me to raise?

Reply:
sure

Reply:
I ended up merging it here.